src/App.vue

```diff
@@ -15,56 +15,56 @@
     <div class="ui inverted top fixed menu">
       <div class="ui container">
         <router-link
-          to="/"
-          class="header item"
+            to="/"
+            class="header item"
         >
           <img
-            class="ui mini image"
-            src="./assets/logo.png"
+              class="ui mini image"
+              src="./assets/logo.png"
           >
           &nbsp;
           Okta-Vue Sample Project
         </router-link>
         <a
-          class="item"
-          v-if="!authState.isAuthenticated"
-          v-on:click="login()"
+            class="item"
+            v-if="authState && !authState.isAuthenticated"
```

Comment:
You should be able to shorten this to `!authState?.isAuthenticated`

Reply:
Confirmed this works in OktaDev Schematics just a few minutes ago. https://github.com/oktadev/schematics/pull/638